tests/Controller/EnMarche/InteractiveControllerTest.php

```diff
@@ -50,11 +49,11 @@ public function testPurchasingPowerAction()
             'purchasing_power[friendFirstName]' => $purchasingPower->friendFirstName = 'Mylène',
             'purchasing_power[friendAge]' => '26',
             'purchasing_power[friendGender]' => $purchasingPower->friendGender = 'female',
-            'purchasing_power[friendPosition]' => '5',
+            'purchasing_power[friendPosition]' => '31',
```

Comment:
Now all the fixtures are always loaded: this is normal that IDs of entities that share the same tables with other entities might change.